core/modules/replica/WorkerServer.cc

```diff
@@ -0,0 +1,129 @@
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "replica/WorkerServer.h"
+
+// System headers
+
+// Third party headers
+#include <boost/bind.hpp>
+
+// Qserv headers
+#include "lsst/log/Log.h"
+#include "replica/Configuration.h"
+#include "replica/ServiceProvider.h"
+#include "replica/WorkerProcessor.h"
+
+namespace {
+
+LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerServer");
```

Comment:
it seems odd to have the underscore in the `_log` name here, it's usually a marker for a private member of a class, no?

Reply:
I have inherited this from other modules in Qserv. This patter is all around. Just do:
fgrep 'LOG_LOGGER _log' core/modules/*/*.cc
core/modules/ccontrol/MergingHandler.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.MergingHandler");
core/modules/ccontrol/UserQueryAsyncResult.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQueryProcessList");
core/modules/ccontrol/UserQueryDrop.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQueryDrop");
core/modules/ccontrol/UserQueryFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQueryFactory");
core/modules/ccontrol/UserQueryFlushChunksCache.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQueryFlushChunksCache");
core/modules/ccontrol/UserQueryProcessList.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQueryProcessList");
core/modules/ccontrol/UserQuerySelect.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQuerySelect");
core/modules/ccontrol/UserQueryType.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQueryType");
core/modules/css/CssAccess.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.css.CssAccess");
core/modules/css/CssConfig.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.css.CssConfig");
core/modules/css/EmptyChunks.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.css.EmptyChunks");
core/modules/css/KvInterfaceImplMem.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.css.KvInterfaceImplMem");
core/modules/css/KvInterfaceImplMySql.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.css.KvInterfaceImplMySql");
core/modules/czar/Czar.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.Czar");
core/modules/czar/CzarConfig.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.CzarConfig");
core/modules/czar/MessageTable.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.MessageTable");
core/modules/global/Bug.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv");
core/modules/mysql/LocalInfile.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.mysql.LocalInfile");
core/modules/mysql/MySqlConfig.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.mysql.MySqlConfig");
core/modules/mysql/MySqlConnection.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.mysql.MySqlConnection");
core/modules/parser/BoolTermFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.parser.BoolTermFactory");
core/modules/parser/dbgParse.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.parser.dbgParse");
core/modules/parser/FromFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.parser.FromFactory");
core/modules/parser/ModFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.parser.ModFactory");
core/modules/parser/ValueExprFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.parser.ValueExprFactory");
core/modules/parser/ValueFactorFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.parser.ValueFactorFactory");
core/modules/parser/WhereFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.parser.WhereFactory");
core/modules/proto/ProtoHeaderWrap.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.parser.ProtoHeaderWrap");
core/modules/proxy/czarProxy.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.proxy.czarProxy");
core/modules/qana/AggregatePlugin.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qana.AggregatePlugin");
core/modules/qana/DuplSelectExprPlugin.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qana.DuplSelectExprPlugin");
core/modules/qana/PostPlugin.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qana.PostPlugin");
core/modules/qana/QservRestrictorPlugin.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qana.QservRestrictorPlugin");
core/modules/qana/QueryMapping.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.query.QueryMapping");
core/modules/qana/RelationGraph.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qana.RelationGraph");
core/modules/qana/ScanTablePlugin.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qana.ScanTablePlugin");
core/modules/qana/TableInfo.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qana.TableInfo");
core/modules/qana/TablePlugin.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qana.TablePlugin");
core/modules/qdisp/Executive.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.Executive");
core/modules/qdisp/JobDescription.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.JobDescription");
core/modules/qdisp/JobQuery.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.JobQuery");
core/modules/qdisp/JobStatus.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.JobStatus");
core/modules/qdisp/MessageStore.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.MessageStore");
core/modules/qdisp/QdispPool.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.QdispPool");
core/modules/qdisp/QueryRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.QueryRequest");
core/modules/qdisp/testQDisp.cc:  LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.testQDisp");
core/modules/qdisp/XrdSsiMocks.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qdisp.XrdSsiMock");
core/modules/qmeta/QMeta.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qmeta.QMeta");
core/modules/qmeta/QMetaMysql.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qmeta.QMetaMysql");
core/modules/qmeta/QMetaSelect.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qmeta.QMetaSelect");
core/modules/qmeta/QMetaTransaction.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qmeta.QMetaTransaction");
core/modules/qproc/IndexMap.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qproc.IndexMap");
core/modules/qproc/QuerySession.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qproc.QuerySession");
core/modules/qproc/SecondaryIndex.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qproc.SecondaryIndex");
core/modules/qproc/TaskMsgFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.qproc.TaskMsgFactory");
core/modules/query/BoolTerm.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.query.BoolTerm");
core/modules/query/OrderByClause.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.query.OrderByClause");
core/modules/query/QueryContext.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.query.QueryContext");
core/modules/query/QueryTemplate.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.query.QueryTemplate");
core/modules/replica/AddReplicaQservMgtRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.AddReplicaQservMgtRequest");
core/modules/replica/Configuration.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.Configuration");
core/modules/replica/ConfigurationFile.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.ConfigurationFile");
core/modules/replica/ConfigurationMySQL.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.ConfigurationMySQL");
core/modules/replica/Controller.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.Controller");
core/modules/replica/CreateReplicaJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.CreateReplicaJob");
core/modules/replica/DatabaseMySQL.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.DatabaseMySQL");
core/modules/replica/DatabaseServices.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.DatabaseServices");
core/modules/replica/DatabaseServicesMySQL.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.DatabaseServicesMySQL");
core/modules/replica/DeleteReplicaJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.DeleteReplicaJob");
core/modules/replica/DeleteRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.DeleteRequest");
core/modules/replica/DeleteWorkerJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.DeleteWorkerJob");
core/modules/replica/FileClient.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.FileClient");
core/modules/replica/FileServer.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.FileServer");
core/modules/replica/FileServerConnection.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.FileServerConnection");
core/modules/replica/FindAllJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.FindAllJob");
core/modules/replica/FindAllRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.FindAllRequest");
core/modules/replica/FindRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.FindRequest");
core/modules/replica/FixUpJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.FixUpJob");
core/modules/replica/GetReplicasQservMgtRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.GetReplicasQservMgtRequest");
core/modules/replica/Job.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.Job");
core/modules/replica/JobController.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.JobController");
core/modules/replica/Messenger.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.Messenger");
core/modules/replica/MessengerConnector.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.MessengerConnector");
core/modules/replica/MoveReplicaJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.MoveReplicaJob");
core/modules/replica/Performance.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.Performance");
core/modules/replica/PurgeJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.PurgeJob");
core/modules/replica/QservGetReplicasJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.QservGetReplicasJob");
core/modules/replica/QservMgtRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.QservMgtRequest");
core/modules/replica/QservMgtServices.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.QservMgtServices");
core/modules/replica/QservSyncJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.QservSyncJob");
core/modules/replica/RebalanceJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.RebalanceJob");
core/modules/replica/RemoveReplicaQservMgtRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.RemoveReplicaQservMgtRequest");
core/modules/replica/ReplicateJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.ReplicateJob");
core/modules/replica/ReplicationRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.ReplicationRequest");
core/modules/replica/Request.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.Request");
core/modules/replica/RequestMessenger.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.RequestMessenger");
core/modules/replica/ServiceManagementRequestBase.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.ServiceManagementRequest");
core/modules/replica/SetReplicasQservMgtRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.SetReplicasQservMgtRequest");
core/modules/replica/StatusRequestBase.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.StatusRequest");
core/modules/replica/StopRequestBase.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.StopRequest");
core/modules/replica/VerifyJob.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.VerifyJob");
core/modules/replica/WorkerDeleteRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerDeleteRequest");
core/modules/replica/WorkerFindAllRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerFindAllRequest");
core/modules/replica/WorkerFindRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerFindRequest");
core/modules/replica/WorkerProcessor.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerProcessor");
core/modules/replica/WorkerProcessorThread.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerProcessorThread");
core/modules/replica/WorkerReplicationRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerReplicationRequest");
core/modules/replica/WorkerRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerRequest");
core/modules/replica/WorkerRequestFactory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerRequestFactory");
core/modules/replica/WorkerServer.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerServer");
core/modules/replica/WorkerServerConnection.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerServerConnection");
core/modules/rproc/InfileMerger.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.rproc.InfileMerger");
core/modules/rproc/ProtoRowBuffer.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.rproc.ProtoRowBuffer");
core/modules/rproc/testInvalidJobAttemptMgr.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.rproc.testInvalidJobAttemptMgr");
core/modules/sql/SqlConnection.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.sql.SqlConnection");
core/modules/tests/qserv-protobuf-assert-version.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.tests.tests_protobuf_version");
core/modules/tests/QueryAnaHelper.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.tests.QueryAnaHelper");
core/modules/util/Command.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.Command");
core/modules/util/ConfigStore.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.ConfigStore");
core/modules/util/Error.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.Error");
core/modules/util/EventThread.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.EventThread");
core/modules/util/InstanceCount.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.InstanceCount");
core/modules/util/Mutex.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.Mutex");
core/modules/util/ThreadPool.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.ThreadPool");
core/modules/util/ThreadPriority.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.ThreadPriority");
core/modules/util/WorkQueue.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.util.WorkQueue");
core/modules/wbase/SendChannel.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wbase.SendChannel");
core/modules/wbase/Task.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wbase.Task");
core/modules/wbase/WorkerCommand.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wbase.WorkerCommand");
core/modules/wconfig/WorkerConfig.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wconfig.WorkerConfig");
core/modules/wcontrol/Foreman.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wcontrol.Foreman");
core/modules/wdb/ChunkResource.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wdb.ChunkResource");
core/modules/wdb/QueryRunner.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wdb.QueryRunner");
core/modules/wdb/QuerySql.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wdb.QuerySql");
core/modules/wdb/SQLBackend.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wdb.ChunkResource");
core/modules/wpublish/AddChunkGroupCommand.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.AddChunkGroupCommand");
core/modules/wpublish/ChunkGroupQservRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.ChunkGroupQservRequest");
core/modules/wpublish/ChunkInventory.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.ChunkInventory");
core/modules/wpublish/ChunkListCommand.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.ChunkListCommand");
core/modules/wpublish/ChunkListQservRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.ChunkListQservRequest");
core/modules/wpublish/GetChunkListCommand.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.GetChunkListCommand");
core/modules/wpublish/GetChunkListQservRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.GetChunkListQservRequest");
core/modules/wpublish/QservRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.QservRequest");
core/modules/wpublish/QueriesAndChunks.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.QueriesAndChunks");
core/modules/wpublish/RemoveChunkGroupCommand.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.RemoveChunkGroupCommand");
core/modules/wpublish/ResourceMonitor.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.ResourceMonitor");
core/modules/wpublish/SetChunkListCommand.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.SetChunkListCommand");
core/modules/wpublish/SetChunkListQservRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.SetChunkListQservRequest");
core/modules/wpublish/TestEchoCommand.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.TestEchoCommand");
core/modules/wpublish/TestEchoQservRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wpublish.TestEchoQservRequest");
core/modules/wsched/BlendScheduler.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wsched.BlendScheduler");
core/modules/wsched/ChunkDisk.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wsched.ChunkDisk");
core/modules/wsched/ChunkTasksQueue.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wsched.ChunkTasksQueue");
core/modules/wsched/GroupScheduler.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wsched.GroupScheduler");
core/modules/wsched/ScanScheduler.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wsched.ScanScheduler");
core/modules/wsched/SchedulerBase.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wsched.ScanScheduler");
core/modules/wsched/testSchedulers.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.wsched.testSchedulers");
core/modules/xrdsvc/ChannelStream.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.ChannelStream");
core/modules/xrdsvc/SsiProvider.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.SsiProvider");
core/modules/xrdsvc/SsiRequest.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.SsiRequest");
core/modules/xrdsvc/SsiService.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.SsiService");
core/modules/xrdsvc/StreamBuffer.cc:LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.StreamBuffer");

Reply:
And I think, the right approach would be this:
::log

This will tell a reader of the code that the variable is defined in the anonymous namespace.

Reply:
Perhaps we should do the Qserv-wide code cleanup to fix this. In a separate ticket.